### ISSUE TITLE
Now restorecon/setfiles require -I to fix mis-labelled files, but fixfiles doesn't have or use -I option

### ISSUE THREAD

**Issue by sourcejedi** [SELinuxProject/selinux#44]
I'm going away for a few days, but I noticed the new behaviour with the digests feature affects `fixfiles`. The optimization can be turned off in restorecon/setfiles using the -I option. But fixfiles has no -I option, and never passes -I to restorecon/setfiles.

>        This script is primarily used to correct the security context database (extended attributes) on filesystems.
>
>       It  can  also  be  run  at  any time to relabel when adding support for new policy, or  just check whether the file contexts are all as you expect.

The "adding support for new policy" case, does not require `-I`. I'm not sure whether omitting `-I` actually helps anything or not in this case... I haven't read up on what use cases the digests are designed to help yet, I just ran into it as an optimization I needed to turn off.

But the other two cases, correcting contexts or checking contexts, do not seem to make much sense without passing `-I` to `setfiles`/`restorecon`.

**Comment by stephensmalley** [comment]
I don't believe this makes any difference currently, because we made the default for setfiles/restorecon to NOT set the security.restorecon_last xattr with the digest, and thus setfiles/restorecon will never skip a filesystem unless the user first ran them with -D to set it in the first place.  But we should likely augment fixfiles to pass through -I if specified.

**Comment by stephensmalley** [comment]
Proposed fix can be found at https://github.com/stephensmalley/selinux/tree/fixfilespassID
